upgrade_tests/helpers/variants.py

```diff
@@ -7,71 +7,51 @@ class VersionError(Exception):
     """Error due to Unsupported Satellite Version"""
 
 
-supported_ver = ['6.1', '6.2']
-from_ver = os.environ.get('FROM_VERSION')
-to_ver = os.environ.get('TO_VERSION')
+_supported_ver = ['6.1', '6.2']
+_FROM_VERSION = os.environ.get('FROM_VERSION')
+_TO_VER = os.environ.get('TO_VERSION')
 
-if from_ver not in supported_ver:
+if _FROM_VERSION not in _supported_ver:
     raise VersionError(
         'Unsupported preupgrade version {} provided for '
-        'entity variants existence tests'.format(from_ver))
+        'entity variants existence tests'.format(_FROM_VERSION))
 
-if to_ver not in supported_ver:
+if _TO_VER not in _supported_ver:
     raise VersionError(
         'Unsupported postupgrade version {} provided for '
-        'entity variants existence tests'.format(to_ver))
-
-
-# The dictionary of entity variants where the key is a component name
-# and value is list of all the component entity variants list those changes
-# during satellite versions
-#
-# Structure of variants directory:
-# {'component_name_e.g_filter,repository':
-#        [
-#        [variant1_6.1, variant1_6.2],
-#        [variant2_6.1, variant2_6.2],
-#        ]
-# }
-#
-# Note: The variants should be listed from 6.1 onwards
-entity_varients = {
-    'filter': [
-        ['lookupkey', 'variablelookupkey'],
-        ['(miscellaneous)', 'foremanopenscap::arfreport'],
-        ['organization', 'katello::subscription'],
-        ['configtemplate', 'provisioningtemplate'],
-        ['view_templates, create_templates, edit_templates, '
-         'destroy_templates, deploy_templates',
-         'view_provisioning_templates, create_provisioning_templates, '
-         'edit_provisioning_templates, destroy_provisioning_templates, '
-         'deploy_provisioning_templates']
-    ]
+        'entity variants existence tests'.format(_TO_VER))
+
+# dict where key is component and value is a dict of properties which name
+# changed from satellite 6.1 to 6.2.
+# On this last dict the key is the property's name on 6.1 and value the name
+#  on 6.2
+_SAT_6_DOT_1_TO_6_DOT_2 = {
```

Comment:
In fact I thought exactly on that case. Think about when 6.3 comes out and a property has its name changed from 6.2. On your way we will have to add something on 6.1 respective matrix column. I can show you how I would handle 6.3 dict, I haven't done because it was not necessary this time, so I kept code simple.